<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FomodScreenshotDialog</class>
 <widget class="QDialog" name="FomodScreenshotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>693</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">background:none;</string>
  </property>
  <layout class="QVBoxLayout" name="backgroundLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="backgroundFrame">
     <property name="styleSheet">
      <string notr="true">QFrame#backgroundFrame{background-color: rgba(0,0,0,85%);}</string>
     </property>
     <layout class="QVBoxLayout" name="mainLayout">
      <item>
       <layout class="QHBoxLayout" name="toolbarLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="slideshowPosition">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel { color:white; }</string>
          </property>
          <property name="text">
           <string notr="true">001/999</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="closeButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/MO/gui/resources/CloseButtonIcon.png</normaloff>:/MO/gui/resources/CloseButtonIcon.png</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="navigateLeft">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true"> QPushButton{border:none; } QPushButton:hover{background-color:rgba(255,255,255,25%); }</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/MO/gui/resources/LeftButtonIcon.png</normaloff>:/MO/gui/resources/LeftButtonIcon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ScaleLabel" name="image">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="navigateRight">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true"> QPushButton{border:none; } QPushButton:hover{background-color:rgba(255,255,255,25%); }</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/MO/gui/resources/RightButtonIcon.png</normaloff>:/MO/gui/resources/RightButtonIcon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="imageTitleLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel { color:white; }</string>
        </property>
        <property name="text">
         <string notr="true">&lt;image name&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="carouselList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableWidget {
    background-color:transparent;
    outline:none;
}

QTableWidget::item {
    selection-background-color:transparent;
    background: none;
}

QScrollBar:horizontal {
    border: none;
    background-color:transparent;
    height: 15px;
    margin: 0px 0px 0px 0px;
}
QScrollBar::handle:horizontal {
    border-radius: 2px;
    background-color: rgba(255,255,255,85%);
    min-width: 20px;
}

QScrollBar::add-line:horizontal {
    width: 0px;
    subcontrol-position: right;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal {
    width: 0px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectColumns</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>80</height>
         </size>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="rowCount">
         <number>1</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <row/>
        <column>
         <property name="text">
          <string notr="true">New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">New Column</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string notr="true">sdf</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string notr="true">ssdf</string>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string notr="true">sf</string>
         </property>
        </item>
        <item row="0" column="3">
         <property name="text">
          <string notr="true">sdf</string>
         </property>
        </item>
        <item row="0" column="4">
         <property name="text">
          <string notr="true">sdf</string>
         </property>
        </item>
        <item row="0" column="5">
         <property name="text">
          <string notr="true">dsfdf</string>
         </property>
        </item>
        <item row="0" column="6">
         <property name="text">
          <string notr="true">sdfsfd</string>
         </property>
        </item>
        <item row="0" column="7">
         <property name="text">
          <string notr="true">sdfsdf</string>
         </property>
        </item>
        <item row="0" column="8">
         <property name="text">
          <string notr="true">sdfsdf</string>
         </property>
        </item>
        <item row="0" column="9">
         <property name="text">
          <string notr="true">sdf</string>
         </property>
        </item>
        <item row="0" column="10">
         <property name="text">
          <string notr="true">sdf</string>
         </property>
        </item>
        <item row="0" column="11">
         <property name="text">
          <string notr="true">sdf</string>
         </property>
        </item>
        <item row="0" column="12">
         <property name="text">
          <string notr="true">sdf</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScaleLabel</class>
   <extends>QLabel</extends>
   <header>scalelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
